Administrator Guide/Managing Volumes.md

```diff
@@ -673,10 +674,10 @@ volume or only on the files which need *healing*.
     *healing*:
 
         # gluster volume heal test-volume info
-        Brick :/gfs/test-volume_0
+        Brick server1:/gfs/test-volume_0
```

Comment:
Nit: Case difference - `server1` vs `Server1`. Also on line 680. It's minor enough, so I'll merge this.

I wonder how the hostname got missed when the doc was written for the first time.